README.md

```diff
@@ -18,9 +18,9 @@ Once that's done, you'll have your account saved in `account.json` in the curren
 
 `manuale` expects the account file to be in your working directory by default, so you'll probably want to make a specific directory to do all your certificate stuff in. Likewise, created certificates get saved in the current path by default.
 
-Next up, verify the domains you want a certificate for with `docker run --rm -ti -v $PWD:/data jgiannuzzi/letsencrypt-manuale authorize [domain]`. This will show you the DNS records you need to create and wait for you to do it. For example, you might do it for `example.com` and `www.example.com`.
+Next up, verify the domains you want a certificate for with `docker run --rm -ti -v $PWD:/data jgiannuzzi/letsencrypt-manuale authorize [domain]`. This will show you the DNS records you need to create and wait for you to do it. For example, you might do it for `example.com` and `www.example.com`. You will need to authorize each hostname you want a certificate for.
 
-Once that's done, you can finally get down to business. Run `docker run --rm -ti -v $PWD:/data jgiannuzzi/letsencrypt-manuale issue example.com www.example.com` to get your certificate. It'll save the key, certificate and certificate with intermediate to the working directory.
+Once that's done, you can finally get down to business. Run `docker run --rm -ti -v $PWD:/data jgiannuzzi/letsencrypt-manuale issue example.com www.example.com` to get your certificates for `example.com` and `www.example.com`. It'll save the key, certificate and certificate with intermediate to the working directory for each hostname that was authorized.
```

Comment:
There is only one certificate created in this example.

Could you change the sentence accordingly?